app/models/course/video.rb

```diff
@@ -51,8 +51,7 @@ class Course::Video < ApplicationRecord
   # @!method self.ordered_by_date_and_title
   #   Orders the videos by the starting date and title.
   scope :ordered_by_date_and_title, (lambda do
-    select('course_videos.*, course_reference_times.start_at, course_lesson_plan_items.title').
-      joins(:lesson_plan_item).
+    joins(:lesson_plan_item).
```

Comment:
Why are we removing the `select`?

Reply:
same as `models/course/assessment.rb`, this select is quite redundant

Reply:
Does this affect the amount of information we fetch?

Reply:
not at all, also by the same reason as the one I've written for the same thing in `models/course/assessment.rb`